app/models/user.rb

```diff
@@ -32,4 +32,9 @@ def ranked?
   def games_needed_to_rank
     games_finished >= GAMES_NEEDED_TO_RANK ? 0 : GAMES_NEEDED_TO_RANK - games_finished
   end
+
+  def self.calculate_win_ratio(won, finished)
```

Comment:
I feel like the code in this method could be inlined in the `win_ratio` instance method and all current usages of this class method could be changed to use the instance method instead. And in both cases (in `StatRetrievalService.find` and `StatRetrievalService.rankings`), the user model is already in a variable, so the code of calling `User.calculate_win_ratio(games_won, games_finished)` is duplicated.